styled/store.js

```diff
@@ -3,6 +3,19 @@ import styled from 'styled-components/native';
 import { Body, ButtonContainer } from '../components/BaseComponents';
 import Colors from '../constants/Colors';
 
+export const MarkerContainer = styled.View`
+  width: 150px
+  display: flex
+  align-items: center
+`;
+export const MarkerStoreName = styled.Text`
```

Comment:
There should be a focused and unfocused version of this. The unfocused version should be poppins-medium,14px, line-height 24. 
![image](https://user-images.githubusercontent.com/21160510/79079349-6436c780-7cc3-11ea-9952-f1b24541272a.png)